src/components/NavBar/NavBar.less

```diff
@@ -0,0 +1,77 @@
+@background-color: #003767;
+@nav-text-color: white;
+@transparent: #ffffff00;
+@breakpoint-width: 879px;
+
+.navbar-container {
+  display: flex;
+  justify-content: space-between;
+  background: @background-color;
+  text-align: left;
+  height: 5rem;
+  padding-left: 2rem;
+  padding-right: 1rem;
+  padding-top: 0.8rem;
+
+  .logo-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    @media (max-width: @breakpoint-width) {
+      padding-top: 0.3rem;
+    }
+
+    .bw-logo {
+      width: 9rem;
+      @media (max-width: @breakpoint-width) {
+        width: 7rem;
+      }
+    }
+
+    .hrf-logo {
+      margin-top: 0.2rem;
+      width: 7rem;
+      @media (max-width: @breakpoint-width) {
```

Comment:
A little confused about these just because I typically keep media queries for the end of the CSS / SCSS / LESS file but I'll communicate with the author about this